src/shared/models/contacts.modal.ts

```diff
@@ -0,0 +1,17 @@
+import { ContactsDTOModel } from './contactsDTO.modal';
+
+export class ContactsModel {
+  public id: number;
+  public type: string;
+  public main: boolean;
+  public phone?: string;
+  public email?: string;
+
+  constructor(contacts: ContactsDTOModel) {
+    this.id = contacts.id;
+    this.type = contacts.type;
+    this.main = contacts.main;
+    this.phone = contacts.phone;
```

Comment:
phone and email aren't required in model, implying that response might not have them

What will happen if your response looks like this:
{
id: 123,
type: 'email',
main: false
}

?

Reply:
We can reseve only one field depends on value of "type" field. Do you think I should add more logic for this dependency?

Reply:
if you received only field, e.g. phone, the other field - email will be undefined, so it may introduce bugs
I guess it will be safer if you add check, whether your field exists or not, and then assign model value

Reply:
Thanks for the advice! I will fix it.